src/Admin/AbstractAdmin.php

```diff
@@ -2919,6 +2921,7 @@ public function canAccessObject($action, $object)
         if (!\is_object($object)) {
             return false;
         }
+
         if (!$this->id($object)) {
```

Comment:
IMO, we must change this condition to `null !== $this->id($object)`.

Reply:
This method is deprecated, we should just let this untouched to avoid conflict.